### PR TITLE
Make user lists sort by full name by default

### LINKED ISSUE #1387: Wrong sorting in user list (firstname lastname)
After an OS3 meeting import into OS4, the participant list is sorted incorrectly. 

Reproduction:
1. Use Export function of OS3 meeting
2. Import data in OS4
3. compare participant listing. --> Wrong sorting. 

What should happen: Correct sorted list after import.

Example: https://testintern.openslides.com/62/participants
OS3 Export file in Files.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #1387

### CODE REVIEW

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Can you give multiple properties to filter for? Full name is still different to first name since it includes title etc. I would like for the "First name" selection to filter by `(first_name, last_name)` to properly display the list.

**Comment by jsangmeister** [comment]
Okay, looks like the `base-sort.service` does not implement this functionality yet, but you extend https://github.com/OpenSlides/openslides-client/blob/d89313bf3fccbeceaee192977098eb784350ebfe/client/src/app/site/base/base-sort.service/base-sort.service.ts#L46 to be able to handle an array of properties and filter by all of them one after another.

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-list/services/participant-list-sort.service/participant-list-sort.service.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ export class ParticipantListSortService extends BaseSortListService<ViewUser> {
      */
     private userSortOptions: OsSortingOption<ViewUser>[] = [
         { property: `full_name`, label: `Full name` },
-        { property: `first_name`, label: `Given name` },
-        { property: `last_name`, label: `Surname` },
+        { property: [`first_name`, `last_name`], label: `Given name` },
+        { property: [`last_name`, `last_name`], label: `Surname` },
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/pages/participants/pages/participant-list/services/participant-list-sort.service/participant-list-sort.service.ts:24]:
```suggestion
        { property: [`last_name`, `first_name`], label: `Surname` },
```
probably you meant this?

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-list/services/participant-list-sort.service/participant-list-sort.service.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ export class ParticipantListSortService extends BaseSortListService<ViewUser> {
      */
     public async getDefaultDefinition(): Promise<OsSortingDefinition<ViewUser>> {
         return {
-            sortProperty: `full_name`,
+            sortProperty: `first_name`,
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/pages/participants/pages/participant-list/services/participant-list-sort.service/participant-list-sort.service.ts:53]:
```suggestion
            sortProperty: [`first_name`, `last_name`],
```

File: client/src/app/site/pages/organization/pages/accounts/pages/account-list/services/account-list-sort.service/account-sort.service.ts [hunk 3]
```diff
@@ -14,8 +14,8 @@ export class AccountSortService extends BaseSortListService<ViewUser> {
 
     private staticSortOptions: OsSortingOption<ViewUser>[] = [
         { property: `full_name`, label: `Full name` },
-        { property: `first_name`, label: `Given name` },
-        { property: `last_name`, label: `Surname` },
+        { property: [`first_name`, `last_name`], label: `Given name` },
+        { property: [`last_name`, `last_name`], label: `Surname` },
```
Comment (jsangmeister) [inline client/src/app/site/pages/organization/pages/accounts/pages/account-list/services/account-list-sort.service/account-sort.service.ts:18]: same here

File: client/src/app/site/pages/organization/pages/accounts/pages/account-list/services/account-list-sort.service/account-sort.service.ts [hunk 4]
```diff
@@ -32,7 +32,7 @@ export class AccountSortService extends BaseSortListService<ViewUser> {
     }
     protected async getDefaultDefinition(): Promise<OsSortingDefinition<ViewUser>> {
         return {
-            sortProperty: `full_name`,
+            sortProperty: `first_name`,
```
Comment (jsangmeister) [inline client/src/app/site/pages/organization/pages/accounts/pages/account-list/services/account-list-sort.service/account-sort.service.ts:35]: same here

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks very good, except for some (i assume) missed places!

**Review by jsangmeister — APPROVED** [review]

**Review by MSoeb — APPROVED** [review]

**Pull request merged**